src/status_im/ui/screens/communities/reorder_categories.cljs

```diff
@@ -59,20 +66,20 @@
   [{:keys [id name community-id]} is-active? drag]
   (let [background-color (if is-active? colors/gray-lighter colors/white)
         category-none?   (string/blank? id)]
-    [:<>
+    [:<> {:style {:flex 1}}
      [quo/separator]
      [rn/view {:accessibility-label :category-item
                :style               (merge styles/category-item
                                            {:background-color background-color})}
-      (if category-none?
+      (if (categories-tab?)
```

Comment:
`(if (not (categories-tab?))`

> Show gray handles only in chats tab

Reply:
or just move ` [icons/icon :main-icons/channel-category {:color colors/gray}]` in else clause of if